Sources/Orbit/Components/SegmentedSwitch.swift

```diff
@@ -0,0 +1,157 @@
+import SwiftUI
+
+/// Showing two choices from which only one can be selected.
+///
+/// Currently SegmentedSwitch allows to have 2 segments.
+/// SegmentedSwitch can be in error state only in unselected state.
+///
+/// - Note: [Orbit definition](https://orbit.kiwi/components/interaction/segmentedswitch/)
+public struct SegmentedSwitch<Value>: View where Value: LosslessStringConvertible & Equatable {
+
+    @Binding private var selection: Value?
+    let label: String
+    let _message: Message?
+    let firstOption: Value
+    let secondOption: Value
+
+    let separatorWidth: CGFloat = .xxxSmall
+    let borderWidth: CGFloat = .xxxSmall
+
+    public var body: some View {
+        FieldWrapper(
+            label,
+            message: message,
+            content: {
+                HStack(spacing: 0) {
+                    segment(title: firstOption)
+                    separator
+                    segment(title: secondOption)
+                }
+                .background(selectionBackground)
+                .background(roundedBackground)
+                .fixedSize(horizontal: false, vertical: true)
+            }
```

Comment:
The whole component should be accessible as a single component.

InputField example:

<img width="828" alt="image" src="https://user-images.githubusercontent.com/35844477/209092475-dde156a1-8649-4ce0-af16-bab1ab647322.png">

Reply:
The hint should only show the message

<img width="236" alt="image" src="https://user-images.githubusercontent.com/35844477/210758241-ea65c984-e842-4017-b2d0-c63ba26d9da9.png">

Reply:
Should be ready as well

Reply:
The `accessibilityHint` still seems to be construct from values as well. It should only show the (optional) message content.

Reply:
Is the change pushed? The `accessibilityHint` still seems to be construct from values as well. It should only show the (optional) message content.

Reply:
I realized it is probably trickier. This would not be accessible anyway, as we do not expose both buttons with their values. Lets keep it as-is for now, please create a follow-up task.

Reply:
Hint is applied on top of `FieldWrapper`
```
   .accessibility(hint: .init(accessibilityHint))
   
   ...
   
    private var accessibilityHint: String {
        "\(message?.description.appending(":") ?? "") \(firstOption)\\\(secondOption)"
    }
```
![image](https://user-images.githubusercontent.com/5494090/211009469-53aca857-9001-461b-bd6c-8e00ff25de3f.png)
![Screenshot 2023-01-06 at 13 03 38](https://user-images.githubusercontent.com/5494090/211009505-00b33234-44f5-4bfa-9082-a3fc972ffef1.png)

Is it wrong? I can create a separate task but what would be the ideal value?

Reply:
The hint should only specify the "message", as in other controls, not values. But a follow-up task is necessary anyway, as this component needs to mimic the accessibility of segmented control.

Reply:
Issue is created here https://github.com/kiwicom/orbit-swiftui/issues/417